src/engine/renderer/glsl_source/cameraEffects_fp.glsl

```diff
@@ -36,9 +38,11 @@ void	main()
 	// calculate the screen texcoord in the 0.0 to 1.0 range
 	vec2 st = gl_FragCoord.st / r_FBufSize;
 
-	vec4 original = clamp(texture2D(u_CurrentMap, st), 0.0, 1.0);
+	vec4 color = clamp(texture2D(u_CurrentMap, st), 0.0, 1.0);
```

Comment:
This would be a nice place for a comment as the double clamping really makes the foolishness of the legacy algorithm apparent.

Reply:
I don't see that place as a good place for such comment, the double clamping would not be a problem if the first one was not too low in a buggy way.

I added a comment in `tr_bsp.cpp` instead in the `R_ColorShiftLightingBytes` function.

Reply:
Oh I was confused and thought that u_LightFactor would be less than 1. Since it is always >= 1 the first clamping has no effect and should be removed.

Reply:
This one is the one actually multiplying the light (and cancelling the cancelation on non-light).


The first clamp in this source file is now removed.